src/locales/en-us/http/index.yaml

```diff
@@ -17,3 +17,9 @@ http:
       certificateExpirationTime: Expiration time
       lastCertificateRegeneration: Last generated
       certificateRegenerations: Regenerations
+      online: 'online'
+      offline: 'offline'
+      partiallyDegraded: 'partially degraded'
+      partially_degraded: 'partially degraded'
+      notAvailable: 'information not available'
+      not_available: 'information not available'
```

Comment:
nit: The thinking here was we would have `http.api.property` and `http.api.value`, is that ok to change up? I don't ever think there would be an instance were we don't know in code whether we are after a `property` or `value` and it could be vaguely useful to have the as separate things.

Reply:
Sure, I changed it.